src/main/java/purpleTeam/MoffatBayLodge/bean/User.java

```diff
@@ -0,0 +1,84 @@
+package purpleTeam.MoffatBayLodge.bean;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+//This is a user Entity bean that maps the user registration table from mysql to the user object in Java.
+//The table name and column name have to match the convention of a snake case. 
+//If it doesn't match snake case and sql exception will thrown by spring.
+
+@Entity
+@Table(name = "user_registration")
+public class User {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "user_id")
+	private int userID;
+
+	@Column(name = "email_address")
+	private String emailAddress;
+
+	@Column(name = "first_name")
+	private String firstName;
+
+	@Column(name = "last_name")
+	private String lastName;
+
+	@Column(name = "phone_number")
+	private String phoneNumber;
+
+	@Column(name = "password")
+	private String password;
+
+	public int getUserID() {
+		return userID;
+	}
+
+	public void setUserID(int userID) {
```

Comment:
Since MySQL handles auto generation of the userId field (i.e AUTO_INCREMENT), I don't believe we should have a setter method for the userId

Reply:
Good catch. Let me update the code